config/rbac/rbac_role.yaml

```diff
@@ -48,6 +78,18 @@ rules:
   - get
   - update
   - patch
+- apiGroups:
+  - constraints.gatekeeper.sh
```

Comment:
I think this is a duplicate of https://github.com/open-policy-agent/gatekeeper/blob/ce52b186de23c18acd574764857bb8408d9941f2/config/rbac/rbac_role.yaml#L37-L48 which already exists.

Reply:
Looks to be. Kubebuilder must not be de-duping. All this code is generated.